adafruit_displayio_layout/widgets/icon_widget.py

```diff
@@ -52,19 +64,67 @@ class IconWidget(Widget, Control):
     :param int max_size: (Optional) this will get passed through to the
      displayio.Group constructor. If omitted we default to
      grid_size width * grid_size height to make room for all (1, 1) sized cells.
+    :param int wheel_initial_value: When using palette animation, this is the initial value
+     of the colorwheel parameter used with ``_pixelbuf.colorwheel``
+    :param int wheel_increment: To add palette animation, set this to the value of
+     how much you want the ``_pixelbuf.colorwheel`` function to increment each time that
+     ``unselected`` is called (default: 0 for no palette animation)
```

Comment:
I think `unselected` was removed in the latest changes.

Reply:
Pushed wrong commit.  The updated commit is now the correct one and this section is deleted.